src/App.js

```diff
@@ -14,11 +15,96 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: ['Jam'],
+  };
+
+  componentDidUpdate() {
+    document.querySelectorAll('li').forEach((element) => {
+      if (this.state.selectedGood.includes(element.textContent)) {
```

Comment:
Let's not use DOM methods while studying react. You can use 'classanmes' library for conditional styles applying